src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -7,7 +7,228 @@ service: Payload CMS
 i18nReady: false
 ---
 
-[Payload](https://payloadcms.com/) is a headless CMS and application framework.
+[PayloadCMS](https://payloadcms.com/) is a headless open-source content management system that can be used to provide content for your Astro project.
+
+## Integrating with Astro
+
+## Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+2. **A MongoDB database** - PayloadCMS will ask you for a MongoDB connection string when creating a new project. You can set one up locally or use [MongoDBAtlas](https://www.mongodb.com/) to host a database on the web for free.
+3. **A PayloadCMS REST API** - Create a [PayloadCMS](https://payloadcms.com/) project and connect it to your MongoDB database during the setup.
+
+### Installing PayloadCMS
+
+1. Follow the [PayloadCMS Installation docs](https://payloadcms.com/docs/getting-started/installation) in order to install a fresh PayloadCMS app. 
+
+:::note[Choosing a template]
+During the PayloadCMS installation, you will be asked if you want to use a template.
+
+If you do not choose a template, only a single collection called "Users" will be generated and you will have to [define a PayloadCMS collection manually](#creating-a-payloadcms-collection) in Astro to use for your content.
+
+Choosing any of the available templates at this step (such as 'blog') automatically generates additional collections for you to use.
+:::
+
+2. Start the PayloadCMS server using `npm run dev` and visit `http://localhost:3000/admin` to enter your PayloadCMS Dashboard for the first time. 
+
+3. Using the Dashboard, create an initial administrator account, which should now be visible as the only entry in your "Users" collection.
+
+#### Creating a PayloadCMS Collection
```

Comment:
```suggestion
### Configuring Astro for your  PayloadCMS collection
```
So now this can be more like:

> Your Payload project template will contain a file called Posts.ts in src/collections/. The example below shows this file for a collection that requires "title"....

and you can skip to that paragraph